tests/js/cases/framework/ajax/Request.test.js

```diff
@@ -0,0 +1,584 @@
+import FakeDom from '../../../helpers/FakeDom';
+
+jest.setTimeout(2000);
+
+describe('Request AJAX library', function () {
+    it('can be setup via a global event', function (done) {
+        FakeDom
+            .new()
+            .addScript([
+                'modules/system/assets/js/framework-next/build/framework.js',
+                'modules/system/assets/js/framework-next/build/framework-js-request.js'
```

Comment:
```suggestion
                'modules/system/assets/js/snowboard/build/framework.js',
                'modules/system/assets/js/snowboard/build/framework-js-request.js'
```
Side note, should the files be `framework.js` or `snowboard.js`?